orca-clouddriver/src/test/java/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/ManifestEvaluatorTest.java

```diff
@@ -58,6 +65,9 @@
   private final TypedString manifestString =
       new TypedString("{'metadata': {'name': 'my-manifest'}}");
 
+  private final TypedString spelManifestString =
+      new TypedString("{'metadata': {'name': '${manifest}'}}");
```

Comment:
this is suppose to be JSON right? JSON doesn't accept single quotes 😞

Reply:
yep, done!